### ISSUE TITLE
Cluster autopscaler chart 9.8.0 not released on helm repo

### ISSUE THREAD

**Issue by TimJones** [kubernetes/autoscaler#3938]
**Which component are you using?**: Cluster Autoscaler Chart

**What version of the component are you using?**: 9.8.0

Component version:

**What k8s version are you using (`kubectl version`)?**: 

<details><summary><code>kubectl version</code> Output</summary><br><pre>
$ kubectl version                    
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2", GitCommit:"f5743093fd1c663cb0cbc89748f730662345d44d", GitTreeState:"clean", BuildDate:"2020-09-16T21:51:49Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.9-eks-d1db3c", GitCommit:"d1db3c46e55f95d6a7d3e5578689371318f95ff9", GitTreeState:"clean", BuildDate:"2020-10-20T22:18:07Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}

</pre></details>

**What environment is this in?**: AWS EKS

**What did you expect to happen?**: Cluster Autoscaler Chart v9.8.0 to be available on the Helm repo.

**What happened instead?**: Only v9.7.0 available on Helm repo.

**How to reproduce it (as minimally and precisely as possible)**:

```
$ helm repo add autoscaler https://kubernetes.github.io/autoscaler                                                                                           
"autoscaler" has been added to your repositories 
$ helm repo update                                                                                                                                           
Hang tight while we grab the latest from your chart repositories...                                                                                           
...Successfully got an update from the "autoscaler" chart repository    
$ helm search repo cluster
NAME                                            CHART VERSION   APP VERSION     DESCRIPTION                                       
autoscaler/cluster-autoscaler                   9.7.0           1.20.0          Scales Kubernetes worker nodes within autoscali...
autoscaler/cluster-autoscaler-chart             2.0.0           1.18.1          Scales Kubernetes worker nodes within autoscali...
```

**Comment by gjtempleton** [comment]
Hey, sorry for the delay, this should now be resolved as #3942 has been merged.

**Comment by gjtempleton** [comment]
Closing as confirmed as resolved:

```bash
helm fetch autoscaler/cluster-autoscaler --version 9.8.0
```

/close
